### DISCUSSION TITLE
Bug: GitHub Actions attempts get confused

### DISCUSSION THREAD

**Discussion by TWiStErRob** [community/community#45610] [Actions]
### Select Topic Area

Bug

### Body

I have a failed attempt, that did not finish due to timeout.

Retry (failed only) running now:
![image](https://user-images.githubusercontent.com/2906988/215261635-b76a5d96-d6b3-4d6f-b145-78bb65ed2cd2.png)

First attempt:
![image](https://user-images.githubusercontent.com/2906988/215261623-3d13a06e-a86a-44cf-a00f-b034f667d874.png)

As you can see there are two status indicators. When I opened the first attempt I saw the steps move just as if I was looking at the retry attempt's logs. It seems the async updates from the server are going to all attempts in the job (if they're open) and not the actual attempt.

After it finished there are even some duplicate steps:
![image](https://user-images.githubusercontent.com/2906988/215261831-fbd9d9a8-729f-499f-b3a1-e9015ededbe5.png)

and when I refresh it jumps weirdly as if there are more duplicates which get deduplicated.

Attempt 1 (first): https://github.com/TWiStErRob/net.twisterrob.cinema/actions/runs/4028195985/jobs/6924812632
Attempt 3 (retry): https://github.com/TWiStErRob/net.twisterrob.cinema/actions/runs/4028195985/jobs/6930610577
(There was an attempt 2 in the middle: https://github.com/TWiStErRob/net.twisterrob.cinema/actions/runs/4028195985/jobs/6925862191, but that is not relevant now.)

**Comment by jge162** [comment]
@TWiStErRob It sounds like there may be a bug in the system you are using to run your job, causing the updates to be applied to all open attempts rather than just the one that is currently running. Additionally, the duplicate steps you are seeing may also be a result of this issue. You should consider submitting a [Bug report](https://support.github.com/contact/bug-report)

**Reply by TWiStErRob** [reply]
Why is there a "bug" category in this Discussions if it's not the way to submit it? 🙄
Anyway, posted to https://support.github.com/ticket/personal/0/1986706 (probably not public, but for reference).

**Reply by jge162** [reply]
developers will see your post here,  I have seen them reply. You do not need to submit a report. I only mentioned as an option if you wished. Thanks again for your detailed post. Have a great weekend

**Reply by TWiStErRob** [reply]
Ah :) anyway, at least they'll surely know about it. I've seen this before, but couldn't pin it, now it was very clear what happened.

**Reply by airtower-luna** [reply]
Sending a bug report is the right way if you want to make sure they see it. :wink:
